modules/mongodb/mongodb.go

```diff
@@ -92,7 +92,7 @@ func WithReplicaSet() testcontainers.CustomizeRequestOption {
 						return err
 					}
 
-					cmd := eval("rs.initiate({ _id: 'rs', members: [ { _id: 0, host: '%s:27017' } ] })", ip)
+					cmd := eval("rs.initiate({ _id: replSetName, members: [ { _id: 0, host: '%s:27017' } ] })", ip)
```

Comment:
Quote replica set name
```suggestion
					cmd := eval("rs.initiate({ _id: '%s', members: [ { _id: 0, host: '%s:27017' } ] })", replSetName, ip)
```